dist/client.d.ts

```diff
@@ -21,7 +21,9 @@ export declare class Client {
     constructor(emitter: Emittery, globalEmitSubscription: () => void);
     connect(): Promise<void>;
     subscribeTicker(symbol: string): void;
+    subscribeTickers(symbol: string[]): void;
```

Comment:
All dist changes should be removed please, it'll be generated automatically by TypeScript compiler

Reply:
Removed!